instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -50,6 +50,7 @@ def query(sql)
               ),
               kind: :client
             ) do |_span, context|
+              sql = sql.dup unless !sql.frozen? || propagator.is_a?(OpenTelemetry::Instrumentation::Trilogy::NoopPropagator)
               propagator.inject(sql, context: context)
               super(sql)
```

Comment:
Should this `freeze` the string again before sending it off it the one it received was already frozen?

Reply:
👍 9ea834e06bc48d59eca272b9c4ebeb2518bca56d